src/server.rs

```diff
@@ -3456,6 +3571,63 @@ errorCounts
         assert_eq!(result, Ok(None));
     }
 
+    // Historically, the completion of a package also brought with it an additional edit
+    // that would import the stdlib module it was referring to. This test asserts that we
+    // don't add it back in, but also serves as documentation that this was a conscious
+    // choice, as the user experience was not good.
+    #[test]
+    async fn test_package_completion_when_it_is_not_imported() {
```

Comment:
Can you add a test for an import that is a full path like `influxdata/influxdb/schema`? I would expect that if a user types `schema` we could offer to import `influxdata/influxdb/schema` for them.